includes/Exception.php

```diff
@@ -450,10 +450,15 @@ public function report(){
  */
 class UserBlockedError extends ErrorPageError {
 	public function __construct( Block $block ){
-		global $wgLang, $wgRequest;
+		global $wgLang, $wgRequest, $wgUser;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$wgUser" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)